lib/chromic_pdf/pdfa/ghostscript_impl.ex

```diff
@@ -7,7 +7,6 @@ defmodule ChromicPDF.GhostscriptImpl do
 
   @default_args [
     "-dQUIET",
-    "-sstdout=/dev/null",
```

Comment:
debugging artifact?

Reply:
True, yes. But that is the reason why ghostscript is very silent when it comes to errors... 
But sure why we want to eat all output (especially since we direct stderr output to stdout). Do you know why we do this?

Reply:
We can improve that in another PR though. I'll create an issue if you like.

Reply:
Probably just wanted to avoid spamming the console with this.. IMO ideally we'd capture the logs and print them in case of an error only.

Reply:
Yes, but the above is only possible without the flag :D